cognitojwt/jwt_async.py

```diff
@@ -12,14 +15,18 @@
 
 @alru_cache(maxsize=1)
 async def get_keys_async(keys_url: str) -> List[dict]:
-    async with aiohttp.ClientSession() as session:
-        async with session.get(keys_url) as resp:
-            response = await resp.json()
-            return response.get('keys')
+    if keys_url.startswith("http"):
+        async with aiohttp.ClientSession() as session:
+            async with session.get(keys_url) as resp:
+                response = await resp.json()
+    else:
+        async with AIOFile(keys_url, 'r') as afp:
+            response = json.loads(await afp.read())
+    return response.get('keys')
 
 
 async def get_public_key_async(token: str, region: str, userpool_id: str):
-    keys_url: str = PUBLIC_KEYS_URL_TEMPLATE.format(region, userpool_id)
+    keys_url: str = os.environ.get('AWS_COGNITO_JWSK_PATH') or PUBLIC_KEYS_URL_TEMPLATE.format(region, userpool_id)
```

Comment:
`os.getenv('AWS_COGNITO_JWSK_PATH')`

Reply:
`os.environ` was used for consistency with the existing tests